playbooks/files/create_dcs_instance.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import openstack
+import sys
+
+conn = openstack.connect()
+
+instance_name = sys.argv[1]
+vpc_id = sys.argv[2]
+subnet_id = sys.argv[3]
+sg_id = sys.argv[4]
+
+instance = conn.dcs.create_instance(
+    name=instance_name,
+    available_zones=["eu-de-03"],
+    capacity=2,
+    engine="Redis",
+    engine_version="3.0",
+    maintain_begin="02:00:00",
+    maintain_end="06:00:00",
+    password="Password.123##",
```

Comment:
if you are not accessing the instance in the scenario, instead of hardcoded pw, use some hash generated pw

Reply:
Please see commit 07e9f2391ef056e1236d343fe271b8d783179e47